src/driver-hidpp20.c

```diff
@@ -343,6 +343,41 @@ hidpp20drv_write_button(struct ratbag_button *button,
 	return -ENOTSUP;
 }
 
+static int hidpp20drv_write_led(struct ratbag_led *led,
+    enum ratbag_led_mode mode,
+    struct ratbag_color color, unsigned int rate,
+    unsigned int bright) {
```

Comment:
do you use some formater?
maybe some style specification or name for this style?